pyEOS/eos.py

```diff
@@ -198,6 +210,29 @@ def replace_config(self, config=None, force=False):
         else:
             raise exceptions.CommandError(result[1]['messages'][0])
 
+    def merge_config(self, config=None):
```

Comment:
Ok, so in anticipation of future changes in the arista implementation (and learning from the difference in how juniper and cisco do it) we will remain with the split for merge and replace right now... yay :)